ods_ci/tests/Resources/CLI/ModelRegistry/samples/postgres/modelregistry_v1alpha1_modelregistry.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: modelregistry.opendatahub.io/v1alpha1
```

Comment:
I think we can avoid having this file duplicated in here and `ods_ci/tests/Resources/CLI/ModelRegistry/samples/mysql/modelregistry_v1alpha1_modelregistry.yaml`

Reply:
@bdattoma This is cloned from https://github.com/opendatahub-io/model-registry-operator/tree/main/config/samples and in future if any changes happened we can change it. I changed my initial git clone approach because of disconnected cluster testing.